### PR TITLE
Review Flags, Fix #53

### LINKED ISSUE #53: Review Flags in Algernon Source
At the moment the Algernon source contains multiple "flags" (self flag: ...) which indicate bugs or open todos. We need to check which of these are real issues, for which GitHub issues exist and whether some of them are already resolved.

### PR DESCRIPTION
**Pull request by robertkirsten**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
+# Algernon Launcher [![GitHub release](https://img.shields.io/github/release/hpi-swa-teaching/Algernon-Launcher.svg?label=small%20release&maxAge=0)](https://github.com/hpi-swa-teaching/Algernon-Launcher/releases/latest) [![Github All Releases](https://img.shields.io/github/downloads/hpi-swa-teaching/Algernon-Launcher/total.svg?maxAge=0)](https://github.com/hpi-swa-teaching/Algernon-Launcher/releases) [![Build Status](https://img.shields.io/travis/hpi-swa-teaching/Algernon-Launcher/master.svg?maxAge=0)](https://travis-ci.org/hpi-swa-teaching/Algernon-Launcher) [![Coverage Status](https://img.shields.io/coveralls/hpi-swa-teaching/Algernon-Launcher/master.svg?maxAge=0)](https://coveralls.io/github/hpi-swa-teaching/Algernon-Launcher?branch=master)
```
Comment (BenBals) [inline README.md]: Is there a reason you added the code coverage badge again? Or is this a merge problem?

**Review by BenBals — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (robertkirsten) [reply README.md]: No seems to be a merge problem

**Review by BenBals — APPROVED** [review]
looks good to me now.

**Comment by BenBals** [comment]
I think we can merge this without further review, as it just removes what amounts to a comment.

**Pull request merged**